tests/test_high_level_io.py

```diff
@@ -56,6 +57,81 @@ def ang_lightcone(ic, lc):
     )
 
 
```

Comment:
Ideally I would also test for gaps and gluts in the data structures themselves, but the different way coevals and lightcones are structured make this difficult.

Lightcones are agnostic to the types of the lightcone fields, so missing fields are ignored and extra ones are simply loaded in.

Coevals on the other hand rely on the output structs, so both missing and extra output fields will raise errors, and changing this behaviour requires adding `safe` flags to all the lower level reading as well.